packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -111,6 +117,24 @@ class AttestationRequestHandler {
     // TODO: Check expiration
   }
 
+  private async verifyPepperIfApplicable(): Promise<void> {
+    if (this.attestationRequest.blindedPhoneNumberSignature) {
```

Comment:
Can you add a metric for whether or not the user provided their phone number? This will help us know when it's safe to force this path

Reply:
the phone number is not an optional parameter, I believe the client has to provide it in the attestation request